README.rst

```diff
@@ -44,14 +44,16 @@ Why PyGMT?
 A beautiful map is worth a thousand words.
 To truly understand how powerful PyGMT is, play with it online on `Binder <https://github.com/GenericMappingTools/try-gmt>`__!
 But if you need some convincing first, watch this `1 hour introduction <https://www.youtube.com/watch?v=SSIGJEe0BIk>`__ to PyGMT!
```

Comment:
```suggestion
But if you need some convincing first, watch this `1 hour introduction <https://youtu.be/SSIGJEe0BIk>`__ to PyGMT!
```

Reply:
Marking this as resolved since there will no longer be a direct link.